subsys/fs/Kconfig.fatfs

```diff
@@ -14,6 +14,15 @@ if FAT_FILESYSTEM_ELM
 menu "ELM FAT file system settings"
 	visible if FAT_FILESYSTEM_ELM
 
+config FS_FATFS_READ_ONLY
+	bool "Read-only support for all volumes"
+	help
+	  The option excludes write code from ELM FAT file system driver;
```

Comment:
The `FS_FATFS_MKFS` option says what it translates to in FatFs.  I think this one should too:

> This option translates to _FS_READONLY within the Zephyr fork of the ELM FAT file system. The option excludes write ....